docs/guides/src/main/server/features.adoc

```diff
@@ -0,0 +1,105 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+
+<@tmpl.guide
+title="Enabling and disabling features"
+summary="Understand how to configure Keycloak to use optional features"
+includedOptions="features features.*">
+
+Keycloak has packed some functionality in features, some of them not enabled by default. These features include features that are in tech preview or deprecated features. In addition there are some features that are enabled by default, but can be disabled if you don't need them for your specific usage scenario.
+
+== List of optional features
+
+[cols="4*", options="header"]
```

Comment:
@stianst fyi: this is why i created https://github.com/keycloak/keycloak/issues/9549 - also the dependency I had to add and the change in Options.java should not be necessary anymore when #9549 is solved.